ambari-server/docs/security/ldap/index.md

```diff
@@ -558,4 +558,4 @@ to take. This is because even though Ambari is managing the LDAP configuration f
 <a name="sync-using-cli"></a>
 ## Synching LDAP users with Ambari Using the CLI
 
-After setting up your LDAP integration, you must synchronize LDAP users and groups with Ambari, using the .`ambari-server sync-ldap [option]` utility. Please read [this guide](https://docs.hortonworks.com/HDPDocuments/HDP3/HDP-3.0.1/ambari-authentication-ldap-ad/content/authe_ldapad_synchronizing_ldap_users_and_groups.html) carefully for further information.
\ No newline at end of file
+After setting up your LDAP integration, you must synchronize LDAP users and groups with Ambari, using the .`ambari-server sync-ldap [option]` utility. Please read [this guide](https://docs.hortonworks.com/HDPDocuments/HDP3/HDP-3.0.1/ambari-authentication-ldap-ad/content/authe_ldapad_synchronizing_ldap_users_and_groups.html) carefully for further information.
```

Comment:
"..the .`ambari-server sync-ldap [option]` utility. "
I believe I added an extra point (.) before '`ambari-server sync-ldap [option]`'; could you please remove it?

Reply:
This is fixed.